core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DsdeHttpDAO.scala

```diff
@@ -1,15 +1,47 @@
 package org.broadinstitute.dsde.rawls.dataaccess
 
-import org.broadinstitute.dsde.rawls.model.UserInfo
-import spray.client.pipelining
-import spray.client.pipelining._
-import spray.http.HttpHeaders.Authorization
+import akka.actor.ActorSystem
+import akka.http.scaladsl.{Http, HttpExt}
+import akka.http.scaladsl.model.{HttpHeader, HttpRequest, ResponseEntity, StatusCodes}
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+import akka.http.scaladsl.unmarshalling.{Unmarshal, Unmarshaller}
+import akka.stream.Materializer
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.rawls.RawlsExceptionWithErrorReport
+import org.broadinstitute.dsde.rawls.model.{ErrorReport, UserInfo}
+import org.broadinstitute.dsde.rawls.util.{HttpClientUtils, HttpClientUtilsStandard}
+
+import scala.concurrent.{ExecutionContext, Future}
 
 /**
  * Created by dvoet on 9/2/15.
  */
-trait DsdeHttpDAO {
-  def addAuthHeader(userInfo: UserInfo): pipelining.RequestTransformer = {
-    addHeader(Authorization(userInfo.accessToken))
+trait DsdeHttpDAO extends LazyLogging {
```

Comment:
akka http client is much different that spray http client... the whole pipeline concept went away. This trait as well as HttpClientUtils are an attempt to make the change less jarring to the rest of the code.